web/src/routes/admin/(resources)/studentGroups/+page.svelte

```diff
@@ -42,8 +42,8 @@
 
   async function deleteGroup(id: string) {
     await deleteStudentGroup(id);
-    studentGroups =
-      (await getStudentGroups({ page: 0, size: 50, sort: ['name,asc'] }).then(({ content }) => content)) || [];
+    const size = await getStudentGroups({ page: 0, size: 1 }).then(({ totalElements }) => totalElements);
```

Comment:
Instead of doing .then it's preferred to destructure the result, so `const { totalElements: size } = await getStudentGroups`